swarm_dial.go

```diff
@@ -619,3 +740,14 @@ func (s *Swarm) IsFdConsumingAddr(addr ma.Multiaddr) bool {
 	_, err2 := first.ValueForProtocol(ma.P_UNIX)
 	return err1 == nil || err2 == nil
 }
+
+func (s *Swarm) IsExpensiveAddr(addr ma.Multiaddr) bool {
+	_, err1 := addr.ValueForProtocol(ma.P_WS)
+	_, err2 := addr.ValueForProtocol(ma.P_WSS)
+	return err1 == nil || err2 == nil
+}
+
+func (s *Swarm) IsRelayAddr(addr ma.Multiaddr) bool {
+	_, err := addr.ValueForProtocol(ma.P_CIRCUIT)
+	return err == nil
+}
```

Comment:
Why are we exposing these? And why are they on the swarm?

Reply:
There is really no good reason to either be exposed or be on the swarm -- I will make them standalone utility funcs.

Reply:
Thanks!

Reply:
done.